relay.go

```diff
@@ -562,6 +589,7 @@ func (r *Relayer) addRelayItem(isOriginator bool, id, remapID uint32, destinatio
 	items := r.inbound
 	if isOriginator {
 		items = r.outbound
+		item.mutatedChecksum = mutatedChecksum
```

Comment:
you can set this in the `relayItem` constructor, since it'll only be set for `isOriginator == true` anyway